pkg/commands/scaffoldcluster.go

```diff
@@ -44,102 +44,81 @@ func sanitizeOpts(opts ScaffoldClusterOpts) ScaffoldClusterOpts {
   # organisation: {{ .Organization }}
 
-  ## The folder to place infrastructure declarations
+  ## Specify the okctl root directory name
   # outputPath: {{ .OutputDirectory }}
 
-## Defines which users can access everything connected to Cognito. Applications connected to Cognito include: ArgoCD UI,
-## Grafana. 
+## Add admin users to your okctl environment
 #users:
 #- email: user.email@emailprovider.org
 
-integrations:
-  ## ArgoCD is a service that watches a repository for Kubernetes charts and ensures the defined resources are running
-  ## as declared in the cluster
-  argoCD: true
-
-  ## Autoscaler automatically adjusts the size of pods and nodes in your cluster depending on load
-  autoscaler: true
-
-  ## AWS Load Balancer Controller handles routing from the internet to your application running inside your okctl
-  ## Kubernetes cluster. If you want your applications and services accessible from the internet, this needs to be
-  ## enabled.
-  awsLoadBalancerController: true
-
-  ## Block storage provides persistent storage for your cluster (Persistent Volumes)
-  blockstorage: true
-
-  ## Cognito is an authentication provider that okctl uses to control access to different resources, like ArgoCD and
-  ## Grafana
-  cognito: true
-
-  ## External DNS handles defining the necessary DNS records required to route traffic to your defined service or 
-  ## application
-  externalDNS: true
-
-  ## External Secrets fetches secrets from external sources and exposes them as native Kubernetes secrets inside the
-  ## cluster 
-  externalSecrets: true
-
-  ## KubePromStack enables Prometheus and Grafana for metrics
-  kubePromStack: true
-
-  ## Loki collects logs and exposes them as a data source in Grafana
-  loki: true
-
-  ## Promtail scrapes logs from pods and feeds them to Loki
-  promtail: true
-
-  ## Tempo collects traces and exposes them as a data source in Grafana. Supports formats like jaeger, zipkin, open 
-  ## telemetry
-  tempo: true
-
-## Defines databases to provision
+#integrations:
+#  # Enable ArgoCD, Git based infrastructure as code (IAC)
+#  argoCD: true
+#
+#  # Enable Autoscaler, automatic scaling of pods and nodes
+#  autoscaler: true
+#
+#  # Enable AWS Load Balancer Controller, automatic provisioning of load balancers
+#  awsLoadBalancerController: true
+#
+#  # Enable Block Storage, persistent storage for pods
+#  blockstorage: true
+#
+#  # Enable Cognito, authentication provider for ArgoCD and Grafana
+#  cognito: true
+#
+#  # Enable ExternalDNS, automatic DNS configuration
+#  externalDNS: true
+#
+#  # Enable External Secrets, external referencing of secrets
+#  externalSecrets: true
+#
+#  # Enable Prometheus and Grafana, for metrics
+#  kubePromStack: true
+#
+#  # Enable Promtail, feeding pod logs to Loki
+#  promtail: true
+#
+#  # Enable Loki, exposing pod logs in Grafana
+#  loki: true
+#
+#  # Enable Tempo, exposing traces in Grafana
+#  tempo: true
+
+## Specify the database(s) for okctl to provision
```

Comment:
I thought this was one long sentence.

I'm tried making more separation to clearly see that "namespace" and "user" is attributes, but I wasn't able to make something I was happy with.

```suggestion
## Specify the database(s) for okctl to provision.
## namespace is for placing database information
## user defines the admin user to provision for operations
```